recipe/meta.yaml

```
@@ -1,27 +1,36 @@
 {% set version = "8.0.0" %}
-{% set sha256 = "95305a886f2c42e9603c034ea684e5c031d9d4222c66ed6d85eb3ae15d631e4b" %}
 
 package:
   name: ipyparallel
   version: {{ version }}
 
 source:
-  fn: ipyparallel-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/i/ipyparallel/ipyparallel-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: 95305a886f2c42e9603c034ea684e5c031d9d4222c66ed6d85eb3ae15d631e4b
 
 build:
-  number: 0
+  number: 1
   noarch: python
+  ignore_run_exports:
+    - nodejs
+  script_env:
+    # don't re-run the nodejs build inside pip install
+    - IPP_DISABLE_JS=1
   entry_points:
     - ipcluster = ipyparallel.cluster.app:main
     - ipcontroller = ipyparallel.controller.app:main
     - ipengine = ipyparallel.engine.app:main
-  script: {{ PYTHON }} -m pip install -vv --no-deps .
-
+  script:
+    - jlpm --ignore-scripts --ignore-optional
+    - jlpm clean:all
+    - jlpm build:prod
+    - {{ PYTHON }} -m pip install -vv --no-deps .
 
 requirements:
   build:
+    - nodejs ==14.*
```

Maybe this would work for relaxing `nodejs` version?

```suggestion
    - nodejs *
```

---

sure, can't hurt